src/field/pokemon.ts

```diff
@@ -1217,6 +1226,16 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     return Math.floor(ret);
   }
 
+  /**
+   * @param target - The {@linkcode} to compare Speed against
+   * @param estimate - If `true`, estimates the target's Speed only based on revealed information
```

Comment:
Technically effective stats are never "revealed information" but i'm not sure how to phrase it better so
```suggestion
   * @param estimate - If `true`, estimates the target's Speed only based on revealed ability information
```